sycl/include/sycl/info/info_desc.hpp

```diff
@@ -204,7 +204,30 @@ template <typename T, T param> struct compatibility_param_traits {};
 #endif
 
 #undef __SYCL_PARAM_TRAITS_SPEC
-
 } // namespace info
+
+#define __SYCL_PARAM_TRAITS_SPEC(Namespace, DescType, Desc, ReturnT, PiCode)   \
+  namespace Namespace::info::DescType {                                        \
```

Comment:
AFAIK this requires C++17 so it shouldn't be used in a core header without a guard. Same goes for other cases of this.

Reply:
Can I know what feature it is ? Maybe there's a workaround for it

Reply:
Having multiple namespaces in a single declaration. The fix should be as simple as

```suggestion
  namespace Namespace { \
  namespace info { \
  namespace DescType {                                        \
```
of course with the corresponding closing brackets.

Reply:
Though you did the right thing, you didn't do it here. 😄

Reply:
Silly me